lms/envs/labster.py

```diff
@@ -34,7 +34,6 @@
 LABSTER_ENDPOINTS = {
     'available_simulations': '',
     'license_validate': '',
-    'voucher_license': '',
```

Comment:
Should we tag devops and ask for provision changes?

Reply:
> Should we tag devops and ask for provision changes?

it's not critical, but we can do when PR is merged.